File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/ContentAddBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class ContentAddBody {
    /// <summary>
    /// File to upload
    /// </summary>
    /// <value>File to upload</value>
    [DataMember(Name="data", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "data")]
    public byte[] Data { get; set; }

    /// <summary>
    /// Filenam to use for upload
    /// </summary>
    /// <value>Filenam to use for upload</value>
    [DataMember(Name="filename", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "filename")]
    public string Filename { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ContentAddBody {\n");
      sb.Append("  Data: ").Append(Data).Append("\n");
      sb.Append("  Filename: ").Append(Filename).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/content_add_body.dart
part of swagger.api;

class ContentAddBody {
  /* File to upload */
  String data = null;
/* Filenam to use for upload */
  String filename = null;

  ContentAddBody();

  @override
  String toString() {
    return 'ContentAddBody[data=$data, filename=$filename, ]';
  }

  ContentAddBody.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    data = json['data'];
    filename = json['filename'];
  }

  Map<String, dynamic> toJson() {
    return {
      'data': data,
      'filename': filename
     };
  }

  static List<ContentAddBody> listFromJson(List<dynamic> json) {
    return json == null ? new List<ContentAddBody>() : json.map((value) => new ContentAddBody.fromJson(value)).toList();
  }

  static Map<String, ContentAddBody> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ContentAddBody>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ContentAddBody.fromJson(value));
    }
    return map;
  }
}
